test/hotspot/jtreg/compiler/vectorapi/VectorRebracket128Test.java

```diff
@@ -59,6 +56,19 @@
     static final VectorSpecies<Byte> bspec128 = ByteVector.SPECIES_128;
     static final VectorSpecies<Short> sspec128 = ShortVector.SPECIES_128;
 
+    static {
+        Thread t = new Thread(() -> {
+            while (true) {
+                try {
+                    System.gc();
```

Comment:
Please, give the following options a try: `-XX:ZCollectionInterval=0.01 -XX:ZFragmentationLimit=0`.
According to ZGC folks, it should force continuous GC cycles w/ ZGC.

Reply:
The original version with option 'CICompilerCount' passed (passed means the bug is not triggered) 5 times in 100 runs, the background gc version passed 8 times in 100 runs. `ZCollectionInterval` passed 44 in 100 runs. 

Explicit trigger gc in the background thread and timer-based gc triggering perform the same thing, it's really strange to    behave differently in triggering the bug. The reason I guess is: the load barrier missing bug will only be triggered when the object is relocated and the pointer in another object is not remapped, which means the time window is very short, different options may pose different execution path (which creates different objects, threads, etc.).

Reply:
Why change the options if the original version was better?